src/otx/algo/classification/huggingface_model.py

```diff
@@ -105,22 +103,6 @@ def _customize_outputs(
             labels=preds,
```

Comment:
Can't `_customize_outputs` be removed?

Reply:
I left this as is to fix the output type as `from transformers.modeling_outputs import ImageClassifierOutput`